api_tests/tsconfig.json

```diff
@@ -10,5 +10,6 @@
         "noUnusedParameters": true,
         "resolveJsonModule": true,
         "esModuleInterop": true
-    }
+    },
+    "include": ["dry", "btsieve", "e2e", "webgui"]
```

Comment:
:question: Shouldn't need this?

> If the "files" and "include" are both left unspecified, the compiler defaults to including all TypeScript (.ts, .d.ts and .tsx) files in the containing directory and subdirectories

https://www.typescriptlang.org/docs/handbook/tsconfig-json.html

Reply:
Indeed I now realised I previously used `tsc` incorrectly.